package/INI/Game Options/Brutal AI.ini

```
@@ -1,18 +1,7901 @@
+;Ensure Global AI Triggers are Not Ignored
+[Basic]
+IgnoreGlobalAITriggers=no
+
+
+
+;Values Needed for new Brutal AI
+[AI]
+BuildPower=NAPOWR,GAPOWR,YAPOWR,NANRCT
+BuildBarracks=NAHAND,GAPILE,YABRCK
+BuildWeapons=GAWEAP,NAWEAP,YAWEAP,RAZER-GAWEAP0,RAZER-NAWEAP0,RAZER-YAWEAP0,RAZER-GAWEAP1,RAZER-NAWEAP1,RAZER-YAWEAP1
+BuildNavalYard=NAYARD,GAYARD,YAYARD,RAZER-GAYARD,RAZER-NAYARD,RAZER-YAYARD
+AlliedBaseDefenses=GAPILL,ATESLA,NASAM,RAZER-NASAM
+SovietBaseDefenses=NALASR,NABNKR,TESLA,NAFLAK,RAZER-NAFLAK
+ThirdBaseDefenses=YAGGUN,YAPSYT,NATBNK,RAZER-YAGGUN,RAZER-YAPSYT
+AIForcePredictionFudge=10,25,80
+BlockagePathDelay=15
+AutocreateTime=.5
+PowerSurplus=300
+WarRatio=.2
+WarLimit=6
+DefenseRatio=.4
+HelipadRatio=.1
+HelipadLimit=2
+AirstripRatio=.12
+AirstripLimit=0
+BaseSizeAdd=6
+AIBaseSpacing=2
```

```suggestion
AIBaseSpacing=1
```

Per https://github.com/CnCNet/cncnet-yr-client-package/pull/376#issuecomment-2438353419 from @Steff833.

---

I set value to 2 in order to prevent the possibility of AI TaskForces grouping up as defensive teams and stopping the AI from building. Can you verify for me if this doesn't happen? Or perhaps a value of 1.5 would work.

---

1.5 looked exactly the same for me. So I guess for this command only whole numbers are working.

---

> So I guess for this command only whole numbers are working.

Ref: https://modenc.renegadeprojects.com/AIBaseSpacing